examples/data-objects/smde/src/index.ts

```diff
@@ -63,18 +36,32 @@ class SmdeContainerFactory extends RuntimeFactoryHelper {
 			[smde.type, Promise.resolve(smde)],
 		]);
 
-		const runtime: ContainerRuntime = await ContainerRuntime.load(
+		const runtime: ContainerRuntime = await ContainerRuntime.loadRuntime({
 			context,
-			registry,
-			// eslint-disable-next-line import/no-deprecated
-			buildRuntimeRequestHandler(
-				// eslint-disable-next-line import/no-deprecated
-				mountableViewRequestHandler(MountableView, [viewRequestHandler]),
-			),
-			undefined, // runtimeOptions
-			undefined, // containerScope
+			registryEntries: registry,
 			existing,
-		);
+			containerScope: context.scope,
+			provideEntryPoint: async (containerRuntime: IContainerRuntime) => {
+				// ISSUE: IContainerRuntime doesn't have methods that expose data stores as IDataStore or
+				// IFluidDataStoreChannel, which expose entryPoint. getRootDataStore returns an IFluidRouter.
+				const dataStore: IFluidDataStoreChannel = (await containerRuntime.getRootDataStore(
+					defaultComponentId,
+				)) as IFluidDataStoreChannel;
+
+				// TODO: better type discovery
+				const smdeDataObject: SmdeDataObject =
+					(await dataStore.entryPoint?.get()) as SmdeDataObject;
+				if (smdeDataObject === undefined) {
+					throw new Error("DataStore did not set its EntryPoint");
+				}
+
+				return new MountableView(
```

Comment:
Here probably want to return something similar to what you're doing for ContainerViewRuntimeFactory (object with both mountable/normal options).  Only want to wrap with a MountableView when crossing a bundle boundary.  Same comment for other examples.